VisionDemo/clients/TFDemo/controllers/page/Start/widgets/ChangeModelButton/onClick.js

```diff
@@ -0,0 +1,8 @@
+	var model = client.getWidget("CurrentModel");
+    if (model.text == 'coco-1.2') {
+        model.text = 'yolo-1.0';
+    } else if (model.text == 'yolo-1.0') {
+        model.text = 'yolov3-1.0';
+    } else {
+        model.text = 'coco-1.2';
+    }
```

Comment:
This click to cycle thing is kinda confusing, and it's going to be a pain to upgrade over time as the most popular models change. Can you change this to be a droplist populated by the tensorflowmodels that you can get by querying on client startup?